app/Entirety/projects/forms.py

```diff
@@ -1,53 +1,84 @@
 from crispy_forms.helper import FormHelper
 from crispy_forms.layout import Submit
 from django import forms
+from users.models import User
 from .models import Project
 
 
 class ProjectForm(forms.ModelForm):
-    def __init__(self, *args, **kwargs):
+    def __init__(self, user, *args, **kwargs):
         super(ProjectForm, self).__init__(*args, **kwargs)
 
         self.helper = FormHelper(self)
 
+        if not user.is_server_admin:
+            self.fields["owner"].disabled = True
```

Comment:
The project admin can not create a project right now. Maybe we can allow empty input of "owner" in create view, and then assign the current user as the owner